### PR TITLE
Adds new foods, both IRL and lore

### PR DESCRIPTION
**Pull request by listerla**
Adds

- V'krexi wraps, as described in Biesel lore
- mac and cheese
- shakshouka
- white chocolate
- A new Biesellite dish, "blue moon"
- A new Europan dish, "deep dive"

All sprited by TomixComics.

### CODE REVIEW

**Comment by listerla** [comment]
!review

**Comment by lavillastrangiato** [comment]
Psionically compelling you to upload the sprites into a drive for the wiki team.

**Comment by listerla** [comment]
looking into the issue

**Review by DreamySkrell — APPROVED** [review]

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -83,6 +83,10 @@
 	drop_sound = 'sound/items/drop/gloves.ogg'
 	pickup_sound = 'sound/items/pickup/gloves.ogg'
 
+/obj/item/trash/snack_bowl/macandcheese
+	name = "empty mac and cheese"
+	icon_state	= "macandcheese"
```
Comment (FluffyGhoster) [inline code/game/objects/items/trash.dm:88]:
```suggestion
	icon_state = "macandcheese"
```

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**